src/Microsoft.AspNetCore.Testing/xunit/FrameworkSkipConditionAttribute.cs

```diff
@@ -33,24 +33,23 @@ private static bool CanRunOnThisFramework(RuntimeFrameworks excludedFrameworks)
                 return true;
             }
 
+#if NET451
             if (excludedFrameworks.HasFlag(RuntimeFrameworks.Mono) &&
                 TestPlatformHelper.IsMono)
             {
                 return false;
             }
 
-            if (excludedFrameworks.HasFlag(RuntimeFrameworks.CLR) &&
-                PlatformServices.Default.Runtime.RuntimeType.Equals("CLR", StringComparison.OrdinalIgnoreCase))
+            if (excludedFrameworks.HasFlag(RuntimeFrameworks.CLR))
             {
                 return false;
             }
-
-            if (excludedFrameworks.HasFlag(RuntimeFrameworks.CoreCLR) &&
-                PlatformServices.Default.Runtime.RuntimeType.Equals("CoreCLR", StringComparison.OrdinalIgnoreCase))
+#else
```

Comment:
It's been broken for a while now: 
(https://github.com/aspnet/Common/blob/dev/src/Microsoft.Extensions.RuntimeEnvironment.Sources/RuntimeEnvironment.cs#L26) and similarly in PlatformAbstractions. https://github.com/aspnet/PlatformAbstractions/blob/dev/src/Microsoft.Extensions.PlatformAbstractions/RuntimeEnvironment.cs#L35.

I guess the right way would be to either detect the CLR we are running on.